.azure-ci.yml

```
@@ -70,42 +112,46 @@ jobs:
             $(BuildScriptArgs)
```

```suggestion
            $(BuildScriptArgs)
            $(_AdditionalBuildArgs)
```
and move the `Build (CodeAnalysis)` step under `- ${{ if ne(variables.runCodeQL3000, 'true') }}:`